_includes/getting-started.html

```diff
@@ -1,106 +1,118 @@
-    <div id="getting-started">
-    <div style="margin-top: auto;" id="box"> 
-      <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
-        with Meshery
-      <p style="font-size: 20px;">Choose a platform and see how easy it is to get up and running with a service mesh.</p></h1>
-    </div>
-  </div> 
-<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -250px;">
-<div class="text editable code">
+<div class="container flex">
+
+  <div class="anchor" style="display:flexbox;" id="box"><a name="getting-started"></a>
```

Comment:
I added the `class="anchor"` tag back, so that the nav menu doesn't overlap the section heading.